docs/content/docs/tasks_usage/traffic_management/_index.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: "Traffic"
-description: "OSM's traffic management stack support two distinct traffic policy modes, namely SMI traffic policy mode and permissive traffic policy mode. The traffic policy mode determines how OSM routes application traffic between pods within the service mesh."
+title: "Traffic Management"
+description: "OSM's traffic management stack support two distinct traffic policy modes, namely SMI traffic policy mode and permissive traffic policy mode. The traffic policy mode determines how OSM routes application traffic between pods within the service mesh. Additionally, ingress and egress functionality allows external access to and from the cluster respectively."
 type: docs
-aliases: ["traffic"]
+aliases: ["traffic management"]
 ---
 
 ## Table of Contents
+- [Egress](./egress.md)
+- [Ingress](./ingress.md)
 - [Permissive traffic policy mode](./permissive_traffic_policy_mode.md)
```

Comment:
Noticed the website sorts the pages in alphabetical order, so followed the same here.